blocks/v2-footer/v2-footer.js

```diff
@@ -32,9 +32,14 @@ function goToTopFunction() {
 }
 
 function addScrollToTopButton(mainEl) {
-  const scrollToTopButton = createElement('button', ['scroll-to-top', 'button'], { title: 'Go to the top of the page' });
-  const icon = createElement('i', ['fa', 'fa-angle-up']);
-  scrollToTopButton.appendChild(icon);
+  const scrollToTopButton = createElement('button', ['v2-scroll-to-top'], { title: 'Go to the top of the page' });
+  // const icon = createElement('i', ['fa', 'fa-angle-up']);
+  // scrollToTopButton.appendChild(icon);
+  const svgIcon = document.createRange().createContextualFragment(`
+    <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path d="M3 11C2.44772 11 2 11.4477 2 12C2 12.5523 2.44772 13 3 13L3 11ZM21.7071 12.7071C22.0976 12.3166 22.0976 11.6834 21.7071 11.2929L15.3431 4.92893C14.9526 4.53841 14.3195 4.53841 13.9289 4.92893C13.5384 5.31946 13.5384 5.95262 13.9289 6.34315L19.5858 12L13.9289 17.6569C13.5384 18.0474 13.5384 18.6805 13.9289 19.0711C14.3195 19.4616 14.9526 19.4616 15.3431 19.0711L21.7071 12.7071ZM3 13L21 13V11L3 11L3 13Z" fill="black"/>
```

Comment:
Replace fill by: `fill="var(--color-icon, #000)"`, in Mack trucks you will be able to update the color of the icon changing the CSS variable in the styles.

Reply:
Fixed

Reply:
please, remove the fill property of the path element `fill="black"` and move `fill="var(--color-icon, #000)"` to the path element.